### PR TITLE
[Documentation] Concepts: Adjustments

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no
| New docs?    | yes
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/adjustments.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+And they can be generally divided into three *groups*: **promotion adjustments**, **shipping adjustments** and **taxes adjustments**.
+
+How to create an Adjustment programmatically?
+---------------------------------------------
+
+The Adjustments alone are a bit useless. They should be created alongside Orders.
+
+As usually get a factory, create a new adjustment.
+Then give it a ``type`` - you can find all the available types on the `AdjustmentInterface <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Core/Model/AdjustmentInterface.php>`_.
+The adjustment needs also the ``amount`` - which is the amount of money that will be** subtracted from the orders total**.
```
Comment (NeverResponse) [inline docs/book/adjustments.rst]: Faulty bolding

File: docs/book/adjustments.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* Tax Adjustments
+
+And they can be generally divided into three *groups*: **promotion adjustments**, **shipping adjustments** and **taxes adjustments**.
+
+How to create an Adjustment programmatically?
+---------------------------------------------
+
+The Adjustments alone are a bit useless. They should be created alongside Orders.
+
+As usually get a factory, create a new adjustment.
```
Comment (NeverResponse) [inline docs/book/adjustments.rst]: `factory and create an adjustment.` **new** is redundant.

File: docs/book/adjustments.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+And they can be generally divided into three *groups*: **promotion adjustments**, **shipping adjustments** and **taxes adjustments**.
+
+How to create an Adjustment programmatically?
+---------------------------------------------
+
+The Adjustments alone are a bit useless. They should be created alongside Orders.
+
+As usually get a factory and create an adjustment.
+Then give it a ``type`` - you can find all the available types on the `AdjustmentInterface <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Core/Model/AdjustmentInterface.php>`_.
+The adjustment needs also the ``amount`` - which is the amount of money that will be **subtracted from the orders total**.
```
Comment (pjedrzejewski) [inline docs/book/adjustments.rst]: That depends: if amount is negative - subtracted, if positive, added.

Comment (pamil) [inline docs/book/adjustments.rst]: The amount is always added, no matter whether it is positive or negative. :)

File: docs/book/adjustments.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+As usually get a factory and create an adjustment.
+Then give it a ``type`` - you can find all the available types on the `AdjustmentInterface <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Core/Model/AdjustmentInterface.php>`_.
+The adjustment needs also the ``amount`` - which is the amount of money that will be **added to the orders total**.
+
+.. note::
+
+   The ``amount`` is always saved in the **base currency**.
+
+Additionally you can set the ``label`` that will be displayed on the order view and whether your adjustment is ``neutral`` -
+**neutral adjustments** do not affect the order's total.
```
Comment (lchrusciel) [inline docs/book/adjustments.rst]: Maybe some explanation of neutral adjustments usage would be nice?

**Comment by pjedrzejewski** [comment]
Thank you Magda!

**Pull request merged**